packages/sanity/src/_createContext/createGlobalScopedContext.ts

```diff
@@ -27,7 +27,9 @@ export function createGlobalScopedContext<ContextType, const T extends ContextTy
       typeof window.navigator !== 'undefined' &&
       window.navigator.userAgent.includes('jsdom'))
   ) {
-    return createContext<ContextType>(defaultValue)
+    const context = createContext<ContextType>(defaultValue)
+    context.displayName = key
```

Comment:
Without `displayName` the context shows up as `UnnamedContext` in profile timelines